### ISSUE TITLE
Timeline Animation/Sequencing System

### ISSUE THREAD

**Issue by Frooxius** [Frooxius/NeosPublic#48]
Implement animation system based on a timeline, where various events, curves, audio files and other elements can be placed to drive other elements in the world.

**Comment by TehTurk** [comment]
To follow from #1420 some notes to add that form of Recording, or Even Exporting Anim files would also be quite handy and add to workflows. While alot of animations have some form of standardized format, this would be beneficial for the future for Neos to Support them in the future. (Unity, Blender, 3DSMax, Maya, etc)

**Comment by Frooxius** [comment]
Yes that would be supported at some point with this system. much like other asset types can be imported and exported.
